Dockerfile

```
@@ -27,7 +27,8 @@ RUN sh -c "$(curl -fsSL https://raw.githubusercontent.com/zdharma/zinit/master/d
 
 # Copy configs into home directory
 ARG FOLDER
-COPY --chown=user "${FOLDER}" /home/user
+ENV XDG_DATA_HOME /home/user/.local/share
+COPY --chown=user ./${FOLDER} /home/user
```

```suggestion
COPY --chown=user ./"$FOLDER" /home/user
```


https://github.com/anordal/shellharden/blob/master/how_to_do_things_safely_in_bash.md#should-i-use-curly-braces